### PR TITLE
Cone Search example to handle situation of RA and DEC column names changing

### LINKED ISSUE #175: Conesearch tutorial occasionally fails because of unpredictable column names
The conesearch tutorial will sometimes fail to run because it may (behind the scenes) change the service it uses to retrieve a table, and different services use different names for RA/Dec. The tutorial assumes "RAJ2000" but it might come back as other names. We should modify the tutorial to be more careful about this.

### PR DESCRIPTION
**Pull request by pllim**
Fix #175 

c/c @adrn @eteq 

I don't know why Jupyter inserted some voodoo incantations. Please let me know if you want me to scrub them off with my good ol' Emacs.

### CODE REVIEW

File: tutorials/vo/conesearch_tutorial.ipynb [hunk 1]
```diff
@@ -127,7 +127,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
```
Comment (pllim) [inline tutorials/vo/conesearch_tutorial.ipynb]: Note: I have no idea why this part changed. I edited the notebook using Jupyter.

**Comment by adrn** [comment]
Could you rebase? (mainly just moving the location of the notebook, I think)

**Comment by pllim** [comment]
It is rebased. Please let me know if you want me to scrub any remaining metadata.

**Comment by pllim** [comment]
Thanks!

**Pull request merged**